src/main/resources/assets/carpet/scripts/distance_rc_1.sc

```diff
@@ -51,5 +64,15 @@ calculate(end_pos)->(
         'w  - Spherical: ', 'w '+spherical+'\n',
         'w  - Cylindrical: ', 'w '+cylindrical+'\n',
         'w  - Manhattan: ', 'w '+manhattan
-    ))
+    ));
+
+    schedule(0,'_draw_line',end_pos);
+);
+
+_draw_line(end_pos)->(
+    if(global_show_line,
+        draw_shape('line',100,'from',global_position,'to',end_pos, 'player',player()),
```

Comment:
This is almost invisible in non-carpet clients (or "not carpet client enabled" servers) since without Carpet in the client, the particles for the line are only generated once per call (probably a bug I should have reported).

That's why other built-in apps usually have a lot lower durations for shapes.

Reply:
Ye, overlay uses delay 15. Problem is then it can get laggy, so I made it refresh each second.